dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/AbstractEventService.java

```diff
@@ -930,15 +920,21 @@ else if ( !ValidationUtils.usernameIsValid( userName ) )
     private OrganisationUnit getOrganisationUnit( IdSchemes idSchemes, String id )
     {
         return organisationUnitCache.get( id,
-            () -> HibernateUtils
-                .initializeProxy( manager.getObject( OrganisationUnit.class, idSchemes.getOrgUnitIdScheme(), id ) ) );
+            () -> manager.getObject( OrganisationUnit.class, idSchemes.getOrgUnitIdScheme(), id ) );
     }
```

Comment:
I added this in previous fix ( https://github.com/dhis2/dhis2-core/pull/8026  ) . But as it didn't fix the issue, so now need to revert it back.